src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -867,8 +867,8 @@ OverlayImpl::getServerInfo()
         Json::Value& validated_ledger = server_info[jss::validated_ledger];
 
         validated_ledger.removeMember(jss::base_fee);
-        validated_ledger.removeMember(jss::reserve_base_xrp);
-        validated_ledger.removeMember(jss::reserve_inc_xrp);
+        validated_ledger.removeMember(jss::reserve_base);
+        validated_ledger.removeMember(jss::reserve_inc);
     }
```

Comment:
@mvadari thank you for that piece of information. Can we confirm this code here is correct? This was just a weird piece in noticed during this PR.